xbmc/utils/EGLUtils.h

```diff
@@ -185,6 +185,7 @@ class CEGLContextUtils final
   bool CreateSurface(EGLNativeWindowType nativeWindow);
   bool CreatePlatformSurface(void* nativeWindow, EGLNativeWindowType nativeWindowLegacy);
   bool CreateContext(CEGLAttributesVec contextAttribs);
+  bool ChooseConfig(EGLint renderableType, EGLint visualId);
```

Comment:
If you make it public, add a check for preconditions (m_eglDisplay) like all other public functions